frontend/public/components/utils/copy-to-clipboard.tsx

```diff
@@ -29,7 +31,7 @@ export const CopyToClipboard: React.FC<CopyToClipboardProps> = React.memo((props
             type="button"
           >
             <CopyIcon />
-            <span className="sr-only">Copy to Clipboard</span>
+            <span className="sr-only">{t('utils~Copy to Clipboard')}</span>
```

Comment:
I think this should be 
```suggestion
            <span className="sr-only">{t('utils~Copy to clipboard')}</span>
```